presto-cache/src/main/java/com/facebook/presto/cache/CacheConfig.java

```diff
@@ -76,4 +83,26 @@ public CacheType getCacheType()
     {
         return cacheType;
     }
+
+    public CacheQuotaScope getCacheQuotaScope()
+    {
+        return cacheQuotaScope;
+    }
+
+    @Config("cache.cache-scope")
+    public void setCacheQuotaScope(CacheQuotaScope cacheQuotaScope)
+    {
+        this.cacheQuotaScope = cacheQuotaScope;
+    }
+
+    public Optional<DataSize> getDefaultCacheQuota()
```

Comment:
`AlluxioCacheConfig.getMaxCacheSize()` can be superseded by this call.

Reply:
What does DefaultCacheQuota represents? Per table quota? If yes then CacheSize(which is worker level cache size for all tables) and this is different.

Reply:
`AlluxioCacheConfig.getMaxCacheSize()` is equivalent to `getDefaultCacheQuota()` with `getCacheQuotaScope == GLOBAL`

Reply:
Can we bring AlluxioCacheConfig.getMaxCacheSize to CacheConfig?  Its not used in the File_Merge but CacheConfig seems like a home for it.

Reply:
It seems different to me, because DefaultCacheQuota can mean different thing depending on  CacheQuotaScope